services/web/server/src/simcore_service_webserver/login/_controller/rest/change.py

```diff
@@ -15,9 +15,8 @@
 from ....utils import HOUR
 from ....utils_rate_limiting import global_rate_limit_route
 from ....web_utils import flash_response
-from ... import _auth_service, _confirmation_service, _confirmation_web
+from ... import _auth_service, _confirmation_web
```

Comment:
stupid Q: what means _confirmation_web ?

Reply:
@sanderegg  This is an example of the `product` domain.

<img width="428" height="356" alt="image" src="https://github.com/user-attachments/assets/19fcb874-5506-4e3c-b29b-bfaeab350b57" />

This corresponds to functionality that belongs to the `web.Application` e.g. events, middlewares or (in this case) helpers for the web api (e.g. that have dependencies with `request: web.REquest`